src/lib/utils/os_utils.h

```diff
@@ -50,6 +50,9 @@ bool running_in_privileged_state();
 */
 uint64_t BOTAN_TEST_API get_cpu_cycle_counter();
 
+long BOTAN_TEST_API get_cpu_total();
+long BOTAN_TEST_API get_cpu_available();
```

Comment:
Both functions should return size_t since the result should never be negative.

(Ideally we would have a `NonZero<T>` guard so as to statically check the result is also != 0, but that's a problem for later)